GOVERNANCE.md

```diff
@@ -0,0 +1,364 @@
+* Informing strengths and weaknesses from a new user perspective 
+(e.g., raising a bug as a GitHub issue, proposing a new feature)
+* Providing moral support (a "thank you" goes a long way).
+
+Users who continue to engage with the project and its community will often
+become more and more involved. Such Users may find themselves becoming
+Contributors, as described in the next section.
+
+### Contributors
+
+Contributors are community members who contribute in concrete ways to the project,
+most often in the form of code and/or documentation. Anyone can become a Contributor,
+and contributions can take many forms, e.g.:
+* Help other users through any of the [communication channels](#communication-channels)
+made for that purpose.
+* Triage GitHub issues.
+* Organize Selenium meetups.
+* Organize and collaborate in Selenium Conferences.
+
+There is no expectation of commitment to the project, no specific skill requirements,
+and no selection process.
+
+Some Contributors might have some basic privileges to the GitHub repos, based on
+their type of contribution (e.g., close an issue after triaging it).
+
+Contributors have read-only access to source code and submit changes via pull
+requests. Contributor pull requests have their contribution reviewed and merged
+by a [Technical Leadership Committee (TLC)](#technical-leadership-committee-tlc) 
+member or a Committer. TLC members and Committers work with Contributors
+to review their code and prepare it for merging.
+
+As Contributors gain experience and familiarity with the project, their profile within,
+and commitment to, the community will increase. At some stage, they may find themselves
+being nominated for the Committer role by an existing Committer or TLC member.
+
+### Project Committers
+
+Committers are community members who have shown that they are committed to the
+continued development of the project through ongoing engagement with the community.
+Committers are given push access to the project's GitHub repos where they
+contribute to.
+
+Committers:
+
+* Should bring their proposals for large changes to the project's code first to
+a GitHub issue, and all relevant committers should be pinged so they can weigh
+in on the discussion if desired. 
+* Debates between committers about whether code should be merged should happen
+in GitHub pull requests to keep the project decision history.
+* In general any committer can review and merge a pull request. Committers
+should only merge code they are qualified to review, which might entail pinging
+another committer who has greater ownership over a specific code area.
+* May label and close issues.
+
+To become a Committer:
+
+* One must have shown a willingness and ability to participate in the project as
+a team player.
+* Committers are expected to be respectful of every community member and to work
+collaboratively in the spirit of inclusion.
+* Have submitted a minimum of 10 qualifying pull requests. Where significant
```

Comment:
This seems a bit arbitrary.  Was this number decided elsewhere?  

What about 

```suggestion
* Have submitted substantive pull requests. Where significant
```

Reply:
The number is a suggestion that I made based on other Governance models that already exist. I strongly think that we need to have a number that can be measured, otherwise things fall into a subjective area. 

Do you think this number makes sense? Should we add `substantive` next to the numer of the number we use?

Reply:
> otherwise things fall into a subjective area.

I think that's one of the best parts of an open source community.  Your contributions for instance, @diemol -  you didn't have 10 PRs ([you had five](https://github.com/diemol?tab=overview&from=2017-12-01&to=2017-12-31) to [SeleniumHQ](https://github.com/seleniumhq)).  But the PRs you *did* have were substantive and I [recommended you](https://groups.google.com/forum/#!msg/selenium-developers/ovmp6B-OrQA/rXOTubxjAgAJ;context-place=forum/selenium-developers) to be on the team :+1:

It is subjective and IMO it should remain that way.  As long as the contributor is also `+1`'d by other people on the team.

Reply:
You are right, I only had 5 PRs and based on this proposal I should not have been added until I reached the 10 PRs. I also believe that having met the maintainers in person, and already maintaining a similar project helped me to get added.

I honestly think we should have a number defined, but we should reach a consensus. We should also define what `substantive` means. And if we only leave `substantive`, does that mean that one single `substantive` PR is enough qualification? This leaves lots of room for interpretation, and a number takes that away.

Nevertheless, as I said, if the rest of us who are watchers of this PR, are ok with leaving `substantive` instead of a fixed number, please just 👍 this comment. If you prefer to have a fixed number, then please 👎 this comment. If possible please add your arguments.

Reply:
This is wrong, working on IDE and Docker containers are part of the official project, just not in the selenium repo. Diego more than qualifies for this position, as does Dave.

I like having an explicit technical requirement, but it's going to be squishy either way since commits/PRs themselves aren't sufficient units of measurement (e.g. if they are all one line comment changes). I think it's fine as is so long as we recognize it as that.

Reply:
I agree with it not be strictly defined... How about wording like -

Have submitted substantive pull requests (for example a minimum of 10).

Reply:
Ok, how about this? Little wordy but more clear I would say.
(Already submitted the change, but putting it here for simplicity)

```
Have submitted sufficient substantive contributions to one or more of the
different projects (IDE, Docker-Selenium, Selenium, Site & Docs). For
technical contributions, enough weight is present and requires little effort to
accept because it is well documented and tested. Normally 10 substantive
contributions are needed to qualify as a candidate to be a committer, but there
could be cases where the contributions are substantial enough that a fewer
amount is also acceptable.
```